### DISCUSSION TITLE
Semi new type of object, slider size changes and adaptable OD,AR,CS,HP in laser editor.

### DISCUSSION THREAD

**Discussion by CelestinSojat** [ppy/osu#15577] [Ideas] [answered]
The new type of object would be a localized spinner which is effected by AR and CS and can be placed anywhere on the beatmap, can have set direction in which it has to be spun indicated with an arrow perhaps as the approach circle, it could also need to be spun for a set amount of spins to be completed or just be like a normal spinner with how much you can spin in the time aloted. The next feature would be the ability to change the size of sliders in a similar way that the shape of a slider is made, the size of the slider could be manipulated via a similar dot system, it could be a checkbox feature you could turn on and off. The last feature i think i saw already mentioned but I really would like for it to be implemented so i am saying it once again + hopefully giving some new ideas on what could be done with it. The ability to change individual CS,OD,AR,HP values, the user interface for this could also be turned on or off with the selected circle/slider/spinner's stats being effected through sliders to change their value for each stat. Also being able to select multiple hitobjects to manipulate at the same time as well as adding another checkbox for a scaling option to make it easier to make patterns that descend or ascend in CS,OD,AR through a ratio that can be made smaller making it a more progressive or bigger to make a more rapid change (not sure i explained this very well but i think you get the idea). These changes would allow mappers extreme amounts of creative freedom to both represent songs far better, to make more creative maps that haven't existed till now and making new kinds of gimmick maps that can elevate the skill expression of players even further. I personally would love for this to be implemented and to see all of the new things mappers would be able to make.

**Comment by peppy** [comment, marked as answer]
Something like this is already supported and planned to be added to the editor at a later stage. Please look out for mechanic additions in the future.

**Reply by CelestinSojat** [reply]
Thank you very much! :D
